test/framework/ownerreference_helpers.go

```diff
@@ -67,12 +69,14 @@ func ValidateOwnerReferencesOnUpdate(ctx context.Context, proxy ClusterProxy, na
 	forceClusterResourceSetReconcile(ctx, proxy.GetClient(), namespace)
 
 	// Check that the ownerReferences have updated their apiVersions to current versions after reconciliation.
+	byf("Check that the ownerReferences are rebuilt as expected")
 	AssertOwnerReferences(namespace, proxy.GetKubeconfigPath(), ownerGraphFilterFunction, assertFuncs...)
```

Comment:
Can you please change all Expect within the Eventually in this func to use the Gomega from here: `Eventually(func(g Gomega) error {`

If I see correctly it's only one in l.129 but please double check

Reply:
@sbueringer PTAL